assets/js/type-defs/contexts.js

```diff
@@ -273,12 +245,10 @@
 /**
  * @typedef {Object} EditorDataContext
  *
- * @property {boolean}                 isEditor           Indicates whether in
- *                                                        the editor context
- *                                                        (true) or not (false).
+ * @property {boolean}                 isEditor           Indicates whether in the editor context.
  * @property {number}                  currentPostId      The post ID being edited.
- * @property {Object}                  previewData        Object containing preview
- *                                                        data for the editor.
+ * @property {Object}                  previewData        Object containing preview  data for the
```

Comment:
And here. :slightly_smiling_face: 
```suggestion
 * @property {Object}                  previewData        Object containing preview data for the
```